src/main/app/jurisdiction/bulletedPointsContent.ts

```diff
@@ -12,69 +12,76 @@ export const enConnectionBulletPointsSummarisedForAllUsers = (
     [JurisdictionConnections.APP_1_APP_2_LAST_RESIDENT]: `both parties to the ${
       isDivorce ? 'marriage' : 'civil partnership'
     } were last habitually resident in England and Wales and one of them continues to reside there.`,
-    [JurisdictionConnections.APP_2_RESIDENT]: `${
-      isJointApplication ? 'applicant 2' : 'the respondent'
-    } is habitually resident in England and Wales`,
-    [JurisdictionConnections.APP_1_RESIDENT_TWELVE_MONTHS]: `${
-      isJointApplication ? 'applicant 1' : 'the applicant'
-    } is habitually resident in England and Wales and has resided there for at least a year immediately prior to the presentation of the application`,
-    [JurisdictionConnections.APP_1_RESIDENT_SIX_MONTHS]: `${
-      isJointApplication ? 'applicant 1' : 'the applicant'
-    } is domiciled and habitually resident in England and Wales and has resided there for at least six months immediately prior to the application`,
-    [JurisdictionConnections.APP_1_APP_2_DOMICILED]: `${isJointApplication ? 'applicant 1' : 'the applicant'} and ${
-      isJointApplication ? 'applicant 2' : 'the respondent'
-    } are both domiciled in England and Wales`,
+    [JurisdictionConnections.APP_2_RESIDENT_SOLE]: 'the respondent is habitually resident in England and Wales',
```

Comment:
```suggestion
    [JurisdictionConnections.APP_2_RESIDENT_SOLE]: 'the respondent is habitually resident in England and Wales.',
```

Reply:
@danielbe11 there shouldn't be any full stops but I've double-checked and I've left a couple in, so I'll remove those